sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -383,7 +384,8 @@ class ResourceNotModifiedError(HttpResponseError):
 class TooManyRedirectsError(HttpResponseError):
     """Reached the maximum number of redirect attempts.
 
-    :param list[str] history: The history of requests made while trying to fulfill the request.
+    :param history: The history of requests made while trying to fulfill the request.
+    :type history: list[~azure.core.pipeline.policies.RequestHistory]
```

Comment:
I'm assuming it was always `RequestHistory` being set here and the typing was just wrong?

Reply:
Right.

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/pipeline/policies/_redirect.py#L132